src/api/routes/controllers/chats.controller.js

```diff
@@ -52,3 +53,39 @@ exports.getChat = async (req, res, next) => {
     });
   }
 };
+
+exports.sendMessage = async (req, res, next) => {
+  const { id } = req.params;
+  const userId = res.locals.user._id;
+  const { profile } = res.locals.user;
+  const { displayName } = res.locals.user;
+  const { message } = req.body;
+  const { createdAt } = req.body;
```

Comment:
구조분해할당으로 깔끔하게 가져올 수 있을 것 같아요!

```suggestion
  const { _id: userId, profile, displayName } = res.locals.user;
  const { message, createdAt } = req.body;
```